components/cop_codegen/src/rpn_function.rs

```diff
@@ -255,6 +256,25 @@ fn parse_arg_type(arg: &FnArg) -> Result<TypePath> {
     Ok(eval_type.clone())
 }
 
+fn check_attr(meta: &[Ident], item_fn: &ItemFn) -> Result<()> {
+    if item_fn.decl.inputs.len() < meta.len() {
+        let arg_num = item_fn.decl.inputs.len();
+        return Err(meta[arg_num]
+            .span()
+            .unwrap()
+            .error("The attrs of `#[rpn_fn]` are more than the params of the function."));
+    }
+    for m in meta {
+        if m != "ctx" && m != "payload" {
+            return Err(meta[0]
```

Comment:
```suggestion
            return Err(m
```
(Don't apply the suggestion on GitHub)